### PR TITLE
Release blog post for ODM 1.1.2 and 1.0.8

### PR DESCRIPTION
**Pull request by alcaeus**
Please don't merge yet, still need to finish changelogs.

### CODE REVIEW

File: site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+=============================================
+
+We are happy to announce the immediate availability of Doctrine MongoDB ODM
+`1.1.2 <https://github.com/doctrine/mongodb-odm/releases/tag/1.1.2>`__ and
+`1.0.8 <https://github.com/doctrine/mongodb-odm/releases/tag/1.0.8>`__.
+
+MongoDB ODM 1.0.8
+-----------------
+
+- calling ``dropCollections()`` in SchemaManager did not drop GridFS collections.
```
Comment (Ocramius) [inline site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst]: `the SchemaManager`.

File: site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+This has been fixed in `#1468 <https://github.com/doctrine/mongodb-odm/pull/1468>`_.
```
Comment (Ocramius) [inline site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst]: I wouldn't use `This has been fixed in` for every entry. Just link it with the issue ID

File: site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- calling ``clear()`` on an uninitialized collection with ``orphanRemoval``
+enabled failed to remove orphaned documents. This has been fixed in
```
Comment (Ocramius) [inline site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst]: Same about `This has been fixed`

File: site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+`#1500 <https://github.com/doctrine/mongodb-odm/pull/1500>`_.
+- Documents with identifiers evaluating to ``false`` (e.g. empty string or 0)
+could not be reference using ``createDbRef()`` in DocumentManager. This edge case
```
Comment (Ocramius) [inline site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst]: Same about `This has been fixed`

File: site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+has been fixed in `#1503 <https://github.com/doctrine/mongodb-odm/pull/1503>`_.
+
+MongoDB ODM 1.1.2
+-----------------
+
+- Querying for referenced objects in ``findBy()`` or ``findOneBy()`` did not work
```
Comment (Ocramius) [inline site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst]: Do the fixes in 1.0.8 also make it to 1.1.2?

File: site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst (hunk 1 above)
Comment (alcaeus) [inline site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst]: Done.

File: site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst (hunk 2 above)
Comment (alcaeus) [inline site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst]: Yes, makes sense.

File: site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+enabled failed to remove orphaned documents. `#1500 <https://github.com/doctrine/mongodb-odm/pull/1500>`_
+- Documents with identifiers evaluating to ``false`` (e.g. empty string or 0)
+could not be reference using ``createDbRef()`` in DocumentManager. `#1503 <https://github.com/doctrine/mongodb-odm/pull/1503>`_
+
+MongoDB ODM 1.1.2
+-----------------
+
+- This release contains the bugfixes outlined for ODM 1.0.8 above
+- Querying for referenced objects in ``findBy()`` or ``findOneBy()`` did not work
+properly due to incorrect preparation of the DbRef objects. `#1481 <https://github.com/doctrine/mongodb-odm/pull/1481>`_
```
Comment (malarzm) [inline site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst]: It's `DBRef` (capital B) IIRC

Comment (alcaeus) [inline site/2016/10/07/doctrine-mongodb-odm-release-1.1.2-and-1.0.8.rst]: Good catch, fixed!

**Pull request merged**